### ISSUE TITLE
Deploy the upload service to EBI AWS

### ISSUE THREAD

**Issue by justincc** [ebi-ait/hca-ebi-dev-team#10]
See [instructions](https://github.com/HumanCellAtlas/upload-service/wiki/Setting-up-a-new-Deployment-of-the-Upload-Service). Sam is happy to work on us with this as necessary.

**Comment by justincc** [comment]
Unblocking since we are going to start hacking away at this ourselves.

**Comment by rdgoite** [comment]
Sam reached out to me last night and told me he has resolved the issue we encountered last time. We'll have another call tonight to realign our deployment with the one he successfully have running.

**Comment by aaclan-ebi** [comment]
Upload service validation now works in prod. However, the error `No module named ‘_cffi_backend’` which keeps happening in the checksum lambda function is still not resolved. I just copied how Rodrey fixed staging last friday ( by exporting the lambda package from HCA AWS account). Looks like there’s a difference in the set of libraries being outputted when you build the python requirements from mac. I noticed the diffs between the .so files for _cffi_backend library in the HCA package and the packages built from my machine.
 
Package built from my machine:                _cffi_backend.cpython-36m-darwin.so
Package from lambda deployed HCA AWS account: _cffi_backend.cpython-36m-x86_64-linux-gnu.so
 
The upload service has a CI/CD pipeline for this and it’s possible that that env is compatible with what libraries works in the lambda function. We might need to setup a linux env to build & deploy the upload service.

To use the HCA CLI, the user must create a config file which would point to the EBI instance of the upload service. It should contain the ff:
```
{
  "upload": {
    "preprod_api_url_template": "https://upload.{deployment_stage}.archive.data.humancellatlas.org/v1",
    "current_area": null,
    "bucket_name_template": "org-hca-data-archive-upload-{deployment_stage}",
    "upload_service_api_url_template": "https://upload.{deployment_stage}.archive.data.humancellatlas.org/v1",
    "production_api_url": "https://upload.archive.data.humancellatlas.org/v1"
  }
}
```
Then change the value of the HCA_CONFIG_FILE env var to point to the location of that config.

`export HCA_CONFIG_FILE=/path/to/file`

e.g.

`export HCA_CONFIG_FILE=~/.config/hca/config.ingest.prod.json`
